src/js/common/schemaform/SaveInProgressErrorPage.jsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+
+import { LOAD_STATUSES, fetchInProgressForm, setFetchFormStatus } from './save-load-actions';
+import SignInLink from '../components/SignInLink';
+import ProgressButton from '../components/form-elements/ProgressButton';
+
+import { updateLogInUrl } from '../../login/actions';
+
+
+// For now, this only handles loading errors, but it could feasibly be reworked
+//  to handle save errors as well if we need it to.
+class SaveInProgressErrorPage extends React.Component {
+  getBackButton = (primary = false) => {
+    const buttonClass = primary ? 'usa-button-primary' : 'usa-button-outline';
+    const goBack = () => {
```

Comment:
I think you could make this a class method (would be slightly more efficient, and I think clearer).